src/main/kotlin/io/sc3/goodies/Registration.kt

```diff
@@ -391,13 +391,25 @@ object Registration {
     val dragonScale = rItem("dragon_scale", BaseItem(itemSettings()
       .maxCount(16)
       .rarity(EPIC)))
-    val popcorn = rItem("popcorn", PopcornItem(itemSettings()
-      .food(PopcornItem.foodComponent)
-      .maxCount(1)))
     val ancientTome = rItem("ancient_tome", AncientTomeItem(itemSettings()
       .maxCount(1)
       .rarity(UNCOMMON)))
 
+    val popcorn = rItem("popcorn", PopcornItem(itemSettings()
+      .food(PopcornItem.foodComponent)
+      .maxCount(1)))
+    val iceCreamVanilla = rIceCreamItem("icecream_vanilla")
+    val iceCreamChocolate = rIceCreamItem("icecream_chocolate")
+    val iceCreamSweetBerry = rIceCreamItem("icecream_sweetberry")
+    val iceCreamNeapolitan = rIceCreamItem("icecream_neapolitan")
+    val iceCreamSpruce = rIceCreamItem("icecream_spruce")
+    val iceCreamMelon = rIceCreamItem("icecream_melon")
+    val iceCreamBeetroot = rIceCreamItem("icecream_beetroot")
+    // This is a bit messy, but a much simpler way to have the sundae be worth more food
+    val iceCreamSundae = rItem("icecream_sundae", IceCreamItem(itemSettings()
```

Comment:
nit: should there be an underscore between `ice` and `cream` because it's two words?
```suggestion
    val iceCreamVanilla = rIceCreamItem("ice_cream_vanilla")
    val iceCreamChocolate = rIceCreamItem("ice_cream_chocolate")
    val iceCreamSweetBerry = rIceCreamItem("ice_cream_sweetberry")
    val iceCreamNeapolitan = rIceCreamItem("ice_cream_neapolitan")
    val iceCreamSpruce = rIceCreamItem("ice_cream_spruce")
    val iceCreamMelon = rIceCreamItem("ice_cream_melon")
    val iceCreamBeetroot = rIceCreamItem("ice_cream_beetroot")
    // This is a bit messy, but a much simpler way to have the sundae be worth more food
    val iceCreamSundae = rItem("ice_cream_sundae", IceCreamItem(itemSettings()
```

Reply:
Oh yeah, this is probably more proper! I didn't think about it that hard to be honest.